### DISCUSSION TITLE
Porting guide

### DISCUSSION THREAD

**Discussion by snoby** [coinfoundry/miningcore#1398] [Q&A] [answered]
Is there any document or FAQ file that describes what is required to port to a new coin?  I've seen that in the blockchain directory there are perhaps some examples, but just wondering if any documents exist?  Do i just have to duplicate has has been accomplished in the block chain directory?  

Thank you,

**Comment by oliverw** [comment, marked as answer]
Not sure why you labeled this a bug, but no there are no documents for that as implementing a new coin or coin family is quite challenging and varies from coin to coin.
